MANIFEST.in

```diff
@@ -1,6 +1,5 @@
 include README.rst
 include versioneer.py
-include pyls_jsonrpc/_version.py
```

Comment:
I think you need to restore this line, now that you readded `_version.py` again.